cli.go

```diff
@@ -0,0 +1,580 @@
+			Usage: "Commit current Artifact. Returns (2) " +
+				"if no update in progress.",
+			Action: func(ctx *cli.Context) error {
+				if len(ctx.Args()) > 0 {
+					return errors.Errorf(
+						errMsgAmbiguousArgumentsGiven,
+						ctx.Args().First())
+				}
+				return runOptions.handleCLIOptions(ctx)
+			},
+		},
+		{
+			Name:  "daemon",
+			Usage: "Run as a daemon.",
+			Action: func(ctx *cli.Context) error {
+				if len(ctx.Args()) > 0 {
+					return errors.Errorf(
+						errMsgAmbiguousArgumentsGiven,
+						ctx.Args().First())
+				}
+				return runOptions.handleCLIOptions(ctx)
+			},
+		},
+		{
+			Name: "install",
+			Usage: "Mender Artifact to install - " +
+				"local file or a `URL`.",
+			ArgsUsage: "<IMAGEURL>",
+			Action: func(ctx *cli.Context) error {
+				runOptions.imageFile = ctx.Args().First()
+				if len(runOptions.imageFile) == 0 {
+					cli.ShowAppHelpAndExit(ctx, 1)
+				}
+				return runOptions.handleCLIOptions(ctx)
+			},
+		},
+		{
+			Name: "rollback",
+			Usage: "Rollback current Artifact. Returns (2) " +
+				"if no update in progress.",
+			Action: func(ctx *cli.Context) error {
+				if len(ctx.Args()) > 0 {
+					return errors.Errorf(
+						errMsgAmbiguousArgumentsGiven,
+						ctx.Args().First())
+				}
+				return runOptions.handleCLIOptions(ctx)
+			},
+		},
+		{
+			Name:  "send-inventory",
+			Usage: "Force inventory update.",
+			Action: func(ctx *cli.Context) error {
+				if len(ctx.Args()) > 0 {
+					return errors.Errorf(
+						errMsgAmbiguousArgumentsGiven,
+						ctx.Args().First())
+				}
+				return updateCheck(
+					exec.Command("kill", "-USR2"),
+					exec.Command("systemctl",
+						"show", "-p",
+						"MainPID", "mender"))
+			},
+		},
+		{
+			Name: "setup",
+			Usage: "Perform configuration setup - " +
+				"'mender setup --help' for command options.",
+			ArgsUsage: "[options]",
+			Action: func(ctx *cli.Context) error {
+				if len(ctx.Args()) > 0 {
+					return errors.Errorf(
+						errMsgAmbiguousArgumentsGiven,
+						ctx.Args().First())
+				}
+				// Handle overlapping global flags
+				if ctx.GlobalIsSet("config") && !ctx.IsSet("config") {
+					runOptions.setupOptions.configPath = runOptions.config
+				} else {
+					runOptions.config = runOptions.setupOptions.configPath
+				}
+				if ctx.IsSet("data") {
+					runOptions.dataStore = ctx.String("data")
+				}
+				if runOptions.Config.ServerCert != "" &&
+					runOptions.setupOptions.serverCert == "" {
+					runOptions.setupOptions.serverCert = runOptions.Config.ServerCert
+				} else {
+					runOptions.Config.ServerCert = runOptions.setupOptions.serverCert
+				}
+				return runOptions.handleCLIOptions(ctx)
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "config, c",
+					Destination: &runOptions.setupOptions.configPath,
+					Value:       defaultConfFile,
+					Usage:       "`PATH` to configuration file."},
+				cli.StringFlag{
+					Name:  "data, d",
+					Usage: "Mender state data `DIR`ECTORY path.",
```

Comment:
Typo: "`DIR`ECTORY"

Reply:
It's actually not a typo, the ampersands uses these characters in the usage string as:
`--data DIR, -d DIR          Mender state data DIRECTORY path.`

Reply:
Nice! I had no idea. Thanks